### PR TITLE
Feature/parser microservice implementation configuration

### PR DESCRIPTION
**Pull request by kogonidze**

### CODE REVIEW

File: Octopus.Core.Parser.WorkerService/Program.cs [hunk 1]
```diff
@@ -14,8 +17,20 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(confBuilder =>
+                {
+                    confBuilder.AddJsonFile("appsettings.json");
+                })
```
Comment (VladiTitov) [inline Octopus.Core.Parser.WorkerService/Program.cs:23]: maybe can create separate json file for each config?

File: Octopus.Core.Parser.WorkerService/Program.cs [hunk 2]
```diff
@@ -14,8 +17,20 @@ public static void Main(string[] args)
...
                 .ConfigureServices((hostContext, services) =>
                 {
+                    var configuration = hostContext.Configuration;
+
+                    services.Configure<ProcessorConfiguration>(configuration.GetSection(nameof(ProcessorConfiguration)));
+
+                    services.Configure<CsvParserConfiguration>(configuration.GetSection(nameof(CsvParserConfiguration)));
+                    services.Configure<JsonParserConfiguration>(configuration.GetSection(nameof(JsonParserConfiguration)));
+                    services.Configure<XmlParserConfiguration>(configuration.GetSection(nameof(XmlParserConfiguration)));
```
Comment (VladiTitov) [inline Octopus.Core.Parser.WorkerService/Program.cs:32]: IOptions - cool!

File: Octopus.Core.Parser.WorkerService/appsettings.json [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+  },
+  "CsvParserConfiguration": {
+    "SkipLines": 6,
+    "FileSeparator": ";"
+  },
+  "XmlParserConfiguration": {
+
+  },
+  "JsonParserConfiguration": {
+
```
Comment (VladiTitov) [inline Octopus.Core.Parser.WorkerService/appsettings.json:17]: add this in .gitignore

**Review by Egorkas — APPROVED** [review]
Well done

File: Octopus.Core.Parser.WorkerService/Program.cs (hunk 1 above)
Comment (kogonidze) [reply Octopus.Core.Parser.WorkerService/Program.cs]: fixed

File: Octopus.Core.Parser.WorkerService/appsettings.json (hunk 3 above)
Comment (kogonidze) [reply Octopus.Core.Parser.WorkerService/appsettings.json]: fixed

**Pull request merged**